hail/python/hail/expr/builders.py

```diff
@@ -12,7 +12,9 @@
 from hail.typecheck import typecheck_method
 
 
-class ConditionalBuilder(object):
+class ConditionalBuilder:
+    """Base class for :class:`.SwitchBuilder` and :class:`.ConditionalBuilder`"""
```

Comment:
I think you mean "SwitchBuilder" and "CaseBuilder"

Reply:
We actually use a different template for Expressions, so this change will not affect those.